website/src/components/home/preview.tsx

```diff
@@ -31,7 +25,7 @@ export default function Preview({ preview }): JSX.Element {
         target="_blank"
         rel="noopener noreferrer"
       >
-        <Video muted loop poster={preview.bgStill}>
+        <Video autoPlay muted loop poster={preview.bgStill}>
```

Comment:
could be worth adding an `engines` limit to `package.json` to prevent anyone else hitting this?

Reply:
@jpdriver this has been added in my last batch of code as we discussed